docs/src/main/sphinx/release/release-352.md

```diff
@@ -0,0 +1,88 @@
+# Release 352 (XX YYY 2020)
+
+## General
+
+* Add support for [`WINDOW` clause](window_clause). ({issue}`651`)
+* Add {func}`version` function. ({issue}`4627`)
+* Allow prepared statement parameters for `SHOW STATS`. ({issue}`6582`)
+* Update tzdata version to 2020d. As a result, queries can no longer reference 
+  `US/Pacific-New` zone, as it has been removed. ({issue}`6660`)
+* Improve performance of queries using {func}`rank()` window function. ({issue}`6333`)
+* Improve performance of {func}`sum` and {func}`avg` for `decimal` types. ({issue}`6951`)
+* Improve join performance. ({issue}`5981`)
+* Improve query planning time. ({issue}`6544`)
+* Fix window and streaming aggregation semantics regarding peer rows. Now peer rows are
+  not distinct from each other, while previously they had to been equal. ({issue}`6472`) 
+* Fix query failure when using element of `array(timestamp(p))` in a complex expression 
+  for `p` greater than 6. ({issue}`6350`)
+
+## Web UI
+
+* Support JWK with certificate chain for OAuth2 authorization. ({issue}`6428`)
+* Optionally verify JWT audience (`aud`) field for OAuth2 Web UI authentication. ({issue}`6501`)
+* Add scopes to `OAuth2` configuration. ({issue}`6580`)
+* Prevent from replay attacks in `OAuth2` by using `nonce` cookie when `openid` scope is requested. ({issue}`6580`)
+
+## JDBC Driver
+
+* Support user impersonation when using password-based authentication. This is configured 
+  using `sessionUser` connection URL parameter. Previously user impersonation was possible
+  only with other authentication mechanisms. ({issue}`6549`)
+* Add OAuth2 authentication support.({issue}`6576`)
+
+## Docker Image
+
+* Remove support for configuration directory `/usr/lib/trino/etc`. The configuration should be provided in `/etc/trino`.
+
+## CLI
+
+* Support user impersonation when using password-based authentication. This is configured
+  using `--session-user` command line parameter. Previously user impersonation was possible 
+  only with other authentication mechanisms. ({issue}`6567`)
+
+## BigQuery Connector
+
+* Fix query failure when executing `count(*)` without `WHERE` clause for views. ({issue}`6706`)
+
+## Hive Connector
+
+* Support reading timestamp with microsecond or nanosecond precision (as configured with 
+  `hive.timestamp-precision` property) nested within `array`, `map` or `struct` data types. ({issue}`5195`)
```

Comment:
We generally support writes too, except for some bugs in Parquet.
Am i correct @jirassimok ?

Reply:
We support writes for ORC, RC binary, and RC text.

After #6622, we can write in everything except Parquet.

Reply:
> We support writes for ORC, RC binary, and RC text (i.e. the formats that have their own writers).

@jirassimok but the "hive native" writers work too, right? so we can write to eg a TEXTFILE as well?

> After #6622, we can write in everything except Parquet.

That PR is tests-only. So after #6622 we _know_ we can write to anything (except Parquet), 
but from user perspective (and so from RNs perspective), we can write even today.
Since that PR is green (or was green earlier), we can declare read/write support already (except Parquet).

Reply:
> "hive native" writers work too, right?

No, because there's two explicit checks for `TIMESTAMP_MILLIS` in `HiveWriteUtils`.